### ISSUE TITLE
FTP support for importing via the command line

### ISSUE THREAD

**Issue by krues8dr** [statedecoded/statedecoded#598]
Another thing I've started on and haven't had time to finish.  Ideally we'll be able to import data from a variety of sources besides on-disk, but as of now we can get data from both American Legal and Municode via FTP, so that takes priority.

**Comment by waldoj** [comment]
So is the idea here that the importer retrieves the data via FTP and puts it into the import directory, instead of the user doing that beforehand?

**Comment by krues8dr** [comment]
Right.  There's enough fiddly bits in file/directory naming on the FTP servers and knowing when the last import was done that it makes sense to keep this in the tool rather than piping things around.

**Comment by krues8dr** [comment]
Moving to future since I no longer have any FTP sites.
